test/cypress/e2e/mapviewer.cy.js

```diff
@@ -0,0 +1,92 @@
+// const datasetIds = [150]
+const datasetIds = [150, 155]
+
+// const apinatomyModels = ['Human Female']
+const apinatomyModels = ['Rat', 'Human Female']
```

Comment:
I think this should be named taxonModels instead.

Reply:
Sure, will update the variable name.